### PR TITLE
:sparkles: New pages and header

### PR DESCRIPTION
**Pull request by KipandaJr**

### CODE REVIEW

**Review by git-marcopitra — APPROVED** [review]

**Comment by git-marcopitra** [comment]
The clicks on nav item are not working correctly on Desktop

File: components/layout/header/index.tsx [hunk 1]
```diff
@@ -9,10 +9,11 @@ import NavBar from './nav-bar';
 const Header: FC = () => (
   <>
     <Box
-      mb="0!important"
+      mb="0 !important"
```
Comment (okusindja) [inline components/layout/header/index.tsx:12]: please, delete this `mb="0 !important"`, it doesn't work and do not affect on the frontend

File: views/pools/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { Box } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import Layout from '@/components/layout';
+
+const Pools: FC = () => {
+  return (
+    <Layout>
+      <Box
```
Comment (okusindja) [inline views/pools/index.tsx:9]: Remove the Box and add Typography

File: views/swap/index.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { Box } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import Layout from '@/components/layout';
+
+const Swap: FC = () => {
+  return (
+    <Layout>
+      <Box
```
Comment (okusindja) [inline views/swap/index.tsx:9]: Remove the Box and add Typography

File: views/error/index.tsx [hunk 4]
```diff
@@ -34,20 +34,25 @@ const Error: FC<ErrorProps> = ({ message, linkGoTo }) => {
...
+        >
           OOPS!
         </Typography>
         <Typography
           mb="1rem"
           textAlign="center"
-          variant="title2"
+          variant="headline"
+          size="large"
           fontFamily="Share Tech Mono"
```
Comment (okusindja) [inline views/error/index.tsx:50]: Do we have this fontFamily in our variants? If yes, you shouldn't add the property here.

File: views/create-token/hero/title.tsx [hunk 5]
```diff
@@ -9,7 +9,8 @@ const Title: FC = () => (
       <Typography
         mr="l"
         as="span"
-        variant="small"
+        variant="body"
+        size="small"
         fontWeight="400"
         lineHeight="4rem"
         fontFamily="Proto"
```
Comment (okusindja) [inline views/create-token/hero/title.tsx:16]: Remove the fontFamily, fontWeight and lineHeight props

File: views/create-token/create-token-form/fixed-supply-toggle.tsx [hunk 6]
```diff
@@ -15,8 +15,7 @@ const FixedSupplyToggle: FC<FixedSupplyToggleProps> = ({
     <>
       <Box display="flex" justifyContent="space-between" color="onSurface">
         <Box>Fixed Supply</Box>
```
Comment (okusindja) [inline views/create-token/create-token-form/fixed-supply-toggle.tsx:17]: Add Typography here, instead of Box

File: utils/toast/index.tsx [hunk 7]
```diff
@@ -18,7 +18,8 @@ export const showTXSuccessToast = async (
     <a target="__black" rel="noreferrer nofollow" href={explorerLink}>
       <Box display="flex" alignItems="center">
         <Typography
-          variant="medium"
+          variant="body"
+          size="medium"
           color="accent"
           textDecoration="underline"
           fontWeight="700"
```
Comment (okusindja) [inline utils/toast/index.tsx:25]: Remove the fontFamily, fontWeight and lineHeight props

**Review by MarioBatalha — APPROVED** [review]

File: components/layout/header/nav-bar.tsx [hunk 8]
```diff
@@ -1,44 +1,38 @@
...
-        px={['s', 's', 's', 'xl']}
-        nActive={{ borderColor: '#0053DB33' }}
-        color={asPath !== '/my-coins' ? 'onSurface' : 'primary'}
-        onClick={() => asPath !== '/my-coins' && push('/my-coins')}
-        borderColor={asPath !== '/my-coins' ? 'transparent' : '#0053DB33'}
-      >
-        My Coins
-      </Box>
+      {MENU_ITEMS.map(({ path, name }) => (
+        <Box
```
Comment (okusindja) [inline components/layout/header/nav-bar.tsx:14]: Fix this. add a Typography as a child in this Box to pass texts inside

**Review by okusindja — CHANGES_REQUESTED** [review]

**Pull request merged**